examples/simple_example/outputs.tf

```
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-output "bucket_name" {
-  description = "The name of the bucket."
-  value       = "${module.folders.bucket_name}"
-}
+output "ids" {
```

The name should indicate that this is a map rather than a list.

```suggestion
output "names_and_ids" {
```

---

Renaming this output, and the impacted test code.